tests/integration/components/editor-with-preview-test.js

```diff
@@ -159,7 +159,7 @@ test('it sets the editor min-height to the editor height when previewing and sti
 
   this.$('.preview').click();
 
-  let style = 'min-height: ' + height + ';';
+  let style = `min-height: ${  height  };`;
```

Comment:
It looks like many of these replacements were made with lots of spacing left over. Not sure if intentional.